src/fsk.c

```diff
@@ -628,10 +628,11 @@ void fsk_demod_core(struct FSK *fsk, uint8_t rx_bits[], float rx_sd[], COMP fsk_
     modem_probe_samp_f("t_f_est",fsk->f_est,M);
     #endif
     float *f_est;
-    if (fsk->freq_est_type)
+    if (fsk->freq_est_type){
         f_est = fsk->f2_est;
-    else
+    }else{
         f_est = fsk->f_est;
+    }
       
```

Comment:
I don't really have any strong conventions here, for single lines lack of braces it can save some whitespace.

Reply:
This was because I was doing some debugging in this area trying to figure out if the mask estimator was working correctly. I can revert if necessary.